dolphinscheduler-spi/src/main/java/org/apache/dolphinscheduler/spi/enums/DbType.java

```diff
@@ -43,7 +43,8 @@ public enum DbType {
     TRINO(12, "trino"),
     STARROCKS(13, "starrocks"),
     AZURESQL(14, "azuresql"),
-    DAMENG(15, "dameng");
+    DAMENG(15, "dameng"),
+    KYUUBI(18, "kyuubi");
```

Comment:
why not set ``` 16 ```?

Reply:
> why not set `16`?

cause 16 is occupied by other datasources implemented by ourselves

Reply:
how about use `17`?